doc/ref_arch/kubernetes/chapters/chapter04.rst

```diff
@@ -543,6 +543,21 @@ the following specifications:
        :cite:t:`multi-net-spec`.
      - gen.ost.01 in :ref:`chapters/chapter02:kubernetes architecture requirements`
      - :cite:t:`anuket-ri2` Chapter 4, section Installation on Bare Metal Infratructure
+   * - ra2.ntw.017
+     - Kubernetes Load Balancer - API
+     - The networking solution deployed within the implementation must include a L4 (TCP/UDP) Load Balancer to steer inbound traffic across the primary interfaces of multiple CNF pods. The solution must support Service type Loadbalancer API and may support the Gateway API additionally.
```

Comment:
```suggestion
     - The networking solution deployed within the implementation must include a L4 (TCP/UDP (excluding QUIC)) Load Balancer to steer inbound traffic across the primary interfaces of multiple CNF pods. The solution must support Service type Loadbalancer API and may support the Gateway API additionally.
```

Reply:
QUIC as a simple UDP session should be supported?